### PR TITLE
76 duplicate terms and fields

### LINKED ISSUE #76: Prevent ability to create duplicate terms and academic fields

### PR DESCRIPTION
**Pull request by rhiannonruth**
Closes #76

### CODE REVIEW

File: spec/controllers/term_controller_spec.rb [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
   describe "GET term" do
     it "gets term by id " do
-      term = FactoryGirl.create(:term)
       get :show, id: term
```
Comment (eripheebs) [inline spec/controllers/term_controller_spec.rb:8]: why is this line deleted? :) surely the test isnt testing anything without it?

Comment (eripheebs) [reply spec/controllers/term_controller_spec.rb:8]: oh i didnt see there was anothe rone above HAHA

Comment (eripheebs) [reply spec/controllers/term_controller_spec.rb:8]: i see now :p

**Comment by eripheebs** [comment]
:shipit:

File: spec/controllers/term_controller_spec.rb (hunk 1 above)
Comment (rhiannonruth) [reply spec/controllers/term_controller_spec.rb:8]: term is created in the let statement so this was repetition, just thought i'd clean it up as i noticed it :)

Comment (rhiannonruth) [reply spec/controllers/term_controller_spec.rb:8]: 👍
